open-craft/sprints
# [BB-3452] Adding ‘Start of New Sprint’ webhook dispatcher to Sprints

Functionality to define member roles (Sprint Planning Manager etc) for the upcoming crafty checklisting feature

**Merge deadline**: None

**Testing instructions**:

1. Checkout this branch
2. Run migrations (docker-compose run django python manage.py migrate)
3. Open the admin panel (localhost:8000/admin)
4. Create a new webhook event called 'new sprint'
5. Create a new webhook and set the webhook url to a temporary request receiver (eg: https://requestbin.com/), link to the webhook event created in the last step
6. Set the HANDBOOK_ROLES_PAGE env variable to 'https://handbook.opencraft.com/en/latest/cells/'
7. Set the FEATURE_CELL_ROLES env variable to True
8. End sprints for each cell and verify that the Webhook receiver receives a list of that cell's members and a list of their roles. Firefighter & Discovery duty should also be in the list as FF-1, FF-2, DD-1, DD-2

**Reviewers**
